src/modules/refresh.js

```diff
@@ -0,0 +1,14 @@
+import Result from './results.js';
+
+const refresh = async () => {
+  const listUl = document.querySelector('.score-table');
+  const results = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/bmBsWnBXvVGZ5GV7fqzY/scores/');
+  const data = await results.json();
+  const arr = data.result;
+  arr.forEach((e) => {
+    const savedRecord = Result.record(e.user, e.score);
+    listUl.innerHTML += savedRecord;
+  });
+};
```

Comment:
- [ ] At the moment, anytime the refresh button is clicked it repopulates the list with new entries. Please kindly ensure it only shows the list of item and not repeat them.